### PR TITLE
Added Terraform 1.x upgrade guide

### PR DESCRIPTION
**Pull request by bwhaley**

### CODE REVIEW

File: _posts/2021-08-02-how-to-update-to-terraform-1-x.adoc
Comment (brikis98) [inline _posts/2021-08-02-how-to-update-to-terraform-1-x.adoc:34]: You need a new line before this. Otherwise, it oddly wraps into the previous line:

Screen Shot 2021-09-06 at 12 20 50 PM

**Review by brikis98 — DISMISSED** [review]
Overall, LGTM! Just one tiny syntax issue to fix.

File: _posts/2021-08-02-how-to-update-to-terraform-1-x.adoc
Comment (bwhaley) [reply _posts/2021-08-02-how-to-update-to-terraform-1-x.adoc]: Good catch. Fixed in 174a3e3.

**Comment by bwhaley** [comment]
Thanks for the review! This will be ready to merge once approved.

**Review by brikis98 — APPROVED** [review]
LGTM!

**Pull request merged**
